examples/spark2/src/main/scala/org/apache/carbondata/examples/DirectSQLExample.scala

```diff
@@ -86,7 +87,7 @@ object DirectSQLExample {
     println("Running SQL on carbon files directly")
     try {
       // 2. run queries directly, no need to create table first
-      sql(s"""select * FROM  carbonfile.`$readPath` limit 10""".stripMargin).show()
+      sql(s"""select * FROM carbon.`$readPath` limit 10""".stripMargin).show()
```

Comment:
Can you add this example(DirectSQLExample) to org.apache.carbondata.examplesCI.RunExamples? and do some check? which can avoid fail when other developer change the code in the future.

Reply:
Done